tailwind.config.js

```diff
@@ -28,19 +28,14 @@ module.exports = {
   theme: {
     extend: {
       screens: {
-        'xl-': { max: '1535px' },
-        'lg-': { max: '1279px' },
-        'md-': { max: '1023px' },
-        'sm-': { max: '767px' },
         'sm+': { min: '640px' },
         'md+': { min: '768px' },
         'lg+': { min: '1024px' },
         'xl+': { min: '1280px' },
-        '2xl+': { min: '1536px' },
-        sm: { min: '640px', max: '767px' },
-        md: { min: '768px', max: '1023px' },
```

Comment:
`md:` is used here. Couldn't find any of the others around:

https://github.com/oxidecomputer/console/blob/0114288974fb9e36bf392277fe59b0de357ec137/app/ui/styles/components/login-page.css#L9-L11

Without it:

<img width="647" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/2df1dffb-a056-443d-8484-c5acb34fb701">

Reply:
Oh, it doesn't error because it still has a definition, the default one. That might not be the reason for the above bug then.

Reply:
Fixed in latest commit – I was applying a gradient to the layout too soon.